ktor-http/common/src/io/ktor/http/URLProtocol.kt

```diff
@@ -58,12 +58,19 @@ public data class URLProtocol(val name: String, val defaultPort: Int) : JvmSeria
          */
         public val SOCKS: URLProtocol = URLProtocol("socks", 1080)
 
+        /**
+         * Data URL protocol for embedded content.
+         *
+         * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.http.URLProtocol.Companion.DATA)
+         */
+        public val DATA: URLProtocol = URLProtocol("data", DEFAULT_PORT)
```

Comment:
This seems redundant

Reply:
let's keep it for consistency

Reply:
As I can see, `URLProtocol` lists only protocols, not any schema, so schemas like `tel`, `about` are not included here. Are you sure `data` should be?

Reply:
Sorry, didn't spot. Let's drop it